Sources/Classes/RSDBPersistentManager.m

```diff
@@ -135,36 +241,23 @@ -(void) createEventsTableWithVersion:(int) version {
 }
 
 - (NSNumber*)saveEvent:(NSString *)message {
+    NSString *insertSQLString = [[NSString alloc] initWithFormat:@"INSERT INTO %@ (%@, %@) VALUES ('%@', %ld) RETURNING %@;", TABLE_EVENTS, COL_MESSAGE, COL_UPDATED, [message stringByReplacingOccurrencesOfString:@"'" withString:@"''"], [RSUtils getTimeStampLong], COL_ID];
     
-    NSString *insertSQLString;
-    if(isReturnClauseSupported) {
-        insertSQLString = [[NSString alloc] initWithFormat:@"INSERT INTO %@ (%@, %@) VALUES ('%@', %ld) RETURNING %@;", TABLE_EVENTS, COL_MESSAGE, COL_UPDATED, [message stringByReplacingOccurrencesOfString:@"'" withString:@"''"], [RSUtils getTimeStampLong], COL_ID];
-    } else {
-        insertSQLString = [[NSString alloc] initWithFormat:@"INSERT INTO %@ (%@, %@) VALUES ('%@', %ld);", TABLE_EVENTS, COL_MESSAGE, COL_UPDATED, [message stringByReplacingOccurrencesOfString:@"'" withString:@"''"], [RSUtils getTimeStampLong]];
-    }
```

Comment:
These lines were removed when SQLCipher was bundled within the SDK and were not added back when the SQLCipher was removed from the SDK.